controllers/admin/AdminShoppingfeedAccountSettings.php

```diff
@@ -363,6 +402,7 @@ public function login($shop_id, $lang_id, $currency_id)
     {
         $username = Tools::getValue('username');
         $password = Tools::getValue('password');
+        $store_id = Tools::getValue('store_id');
```

Comment:
store_id should be an integer according to the object model definition.
You probably should cast it or add a validation before using it.

Reply:
the modifications were added